src/msg/async/ProtocolV1.cc

```diff
@@ -315,6 +315,17 @@ void ProtocolV1::write_event() {
     auto start = ceph::mono_clock::now();
     bool more;
     do {
+
+      if (connection->outgoing_bl.length() > 0) {
```

Comment:
```suggestion
      if (connection->is_queued()) {
```

Reply:
thanks for reminder the related function
done with latest versions